libyara/stopwatch.c

```diff
@@ -48,6 +48,7 @@ uint64_t yr_stopwatch_elapsed_us(
 
   QueryPerformanceCounter(&li);
 
+  if (sw->frequency.QuadPart <= 0) return 0;
```

Comment:
This is just avoiding the crash, but is not solving the problem because all calls to `yr_stopwatch_elapsed_us` are going to return 0. My guess is that this happens with the 32bit YARA, is that correct? Did you reproduce the issue?

Reply:
From the name `RogueKiller64.exe` I would say that you are running a 64bit YARA, it would be very useful if you look at `sw->frequency` and tell me what you see. `sw->frequency` is a `LARGE_INTEGER` which is something like: 

```C
typedef union _LARGE_INTEGER {  
    struct {
        DWORD LowPart;  
        LONG  HighPart; 
    };
    LONGLONG QuadPart;
} LARGE_INTEGER;
```

It's very weird that QueryPerformanceFrequency returns 0.

Reply:
Here's full context: 
![image](https://user-images.githubusercontent.com/9531438/53638278-09a8ec00-3c26-11e9-8c47-24e71b033862.png)

Reply:
Sorry for the delay, I'll take a look and let you know. 
I just need to accumulate more crash dumps with that bug.

Reply:
Hello, sorry for the delay, that's the issue I think:
In yr_scanner_scan_mem_blocks (the frame above), I have ret = 1 so I think it exists on line 385 and skips initialization. Is this fixed yet ?

Reply:
![image](https://user-images.githubusercontent.com/9531438/64673833-8cc9bb00-d46f-11e9-9066-8aa0363b9634.png)